charms/istio-gateway/metadata.yaml

```diff
@@ -5,6 +5,7 @@ description: |
   https://istio.io/latest/docs/tasks/traffic-management/ingress/
   https://istio.io/latest/docs/tasks/traffic-management/egress/
 min-juju-version: "2.9.0"
+series: [kubernetes]
```

Comment:
I thought this would change the charm to a podspec charm, which would have prevented lightkube from working at all (due to RBAC).  Have you tested with rbac enabled?

The CI I think would fail here, but there's conflicts with main and it wont run until they're resolved

Reply:
Okay, I will remove it and retry

Reply:
I retried without ```series: [kubernetes]``` and it errors with:
```
$ juju deploy ./istio.yaml --trust
Located charm "dex-auth" in charm-hub, channel 2.28/stable
ERROR cannot deploy bundle: min-juju-version in metadata v2 not valid
```
Bundle used: https://pastebin.canonical.com/p/kw7nVtQGJS/

I will keep that setting.

Reply:
I retried without ```series: [kubernetes]``` and it errors with:
```
$ juju deploy ./istio.yaml --trust
Located charm "dex-auth" in charm-hub, channel 2.28/stable
ERROR cannot deploy bundle: min-juju-version in metadata v2 not valid
```
Bundle used: https://pastebin.canonical.com/p/kw7nVtQGJS/

I will keep that setting.

Reply:
It's very likely that you were building istio from source on a HEAD that had [this](https://github.com/canonical/istio-operators/commit/e8285038956a38bc4dd1de666ba001e6052e4ef4) commit. At the moment we didn't know we had to use `assumes` instead of `min-juju-version`. I suggest you rebase your branch, leave `metadata.yaml` as it is right now in master and try building and deploying istio again. If the problem persists, please let us know, but this shouldn't happen.